templates/csharp/sso-tab/Components/Graph.razor

```diff
@@ -49,7 +52,18 @@
     private async Task ShowProfile()
     {
         IsLoading = true;
-        var graph = GetGraphServiceClient();
+
+        var config = Configuration.Get<ConfigOptions>();
+        var tenantId = config.TeamsFx.Authentication.OAuthAuthority.TrimStart("https://login.microsoftonline.com/".ToCharArray());
+        AccessToken ssoToken = await teamsUserCredential.GetTokenAsync(new TokenRequestContext(null), new CancellationToken());
+        var tokenCredential = new OnBehalfOfCredential(
```

Comment:
Overall looks good to me. One only question is: if user does not consent MS Graph permission, and OBO failed, will the app correctly handle the exception and ask user login? Not sure what exception will OBOCredential throw, and whether we need to catch the new exception in the login logic.